carbonmark-api/src/routes/projects/projects.types.ts

```diff
@@ -1,3 +1,41 @@
 // export type PoolProject = Project & CarbonOffset;
 
-export type PoolProject = { isPoolProject?: boolean };
+import {
+  Activity,
+  Category,
+  Country,
+  FindProjectsQuery,
+} from "src/.generated/types/marketplace.types";
+import { FindCarbonOffsetsQuery } from "src/.generated/types/offsets.types";
+import { Nullable } from "../../../../lib/utils/typescript.utils";
```

Comment:
Are you able to use `@klimadao/lib/utils`?
This defies our current conventions in the monorepo (using the alias & explicit barrel file exports). I'm surprised it actually works, probably because it's only a type and gets removed at build time...